ocs_ci/utility/utils.py

```diff
@@ -352,21 +352,44 @@ def custom_ceph_config(suite_config, custom_config, custom_config_file):
     return full_custom_config
 
 
+def mask_secrets(plaintext, secrets):
+    """
+    Replace secrets in plaintext with asterisks
+
+    Args:
+        plaintext (str): The plaintext to remove the secrets from
+        secrets (list): List of secret strings to replace in the plaintext
+
+    Returns:
+        (str): The censored version of plaintext
+
+    """
+    for secret in secrets:
+        plaintext = plaintext.replace(secret, '*' * 5)
```

Comment:
replace 5 with len(secret) if that's the intention

Reply:
len(secret) might become a security issue, so I and Petr agreed it'd be best to mask it with a static length

Reply:
I suggested in some previous comment to not to do that and just keep with some static length here which doesn't tell anything about length of secret Raz

Reply:
ack